### ISSUE TITLE
Bug: Embeded links are not displayed

### ISSUE THREAD

**Issue by mariopavlov** [valentine195/obsidian-admonition#130]
Hello,

From the last update (not sure if of Obsidian or Admonition I noticed that embed blocks are not rendered anymore.

Here is the example (in view mode):
![image](https://user-images.githubusercontent.com/9350618/143236188-bb7c161c-449b-46d6-984e-10c0a43715ff.png)

And here is how it looks in edit mode:
![image](https://user-images.githubusercontent.com/9350618/143236265-ff7a92f1-1583-48aa-bca4-889ff3e937ba.png)

Besides the latest updates of Obsidian and Admonition, I also installed CodeMirror Options (not sure if it is relevant).

**Comment by valentine195** [comment]
Hmm, looks like this is from an Obsidian update - went back a few versions on Admonitions and they're all showing the same behavior. I assume you are on Obsidian 0.13+? 

My guess is their markdown renderer logic was updated. Let me play around with it.

**Comment by mariopavlov** [comment]
Yes, I'm using **0.13.5** with **Experimental Live View** disabled.

**Comment by valentine195** [comment]
I will have to work with the Obsidian devs on how to solve this. Seems like the embed loading logic was removed from the exposed markdown rendering function.

**Comment by mariopavlov** [comment]
OK nice, meanwhile I enabled again CodeMirror Options and I saw that in edit mode I can see again the embed links inside admonition

**Comment by Mara-Li** [comment]
I have also the same issue, for embed block and image : 
![image](https://user-images.githubusercontent.com/30244939/144036410-e3fb950e-31f2-40a8-be4e-6ed6cd66999b.png)

**Comment by valentine195** [comment]
This is fixed in the latest Obsidian Insider release.
